pkg/graph/resolvers/model_plan.go

```diff
@@ -348,7 +365,7 @@ func ModelPlanCollection(logger *zap.Logger, principal authentication.Principal,
 	case model.ModelPlanFilterCollabOnly:
 		modelPlans, err = store.ModelPlanCollectionCollaboratorOnly(logger, false, principal.Account().ID)
 	case model.ModelPlanFilterWithCrTdls:
-		modelPlans, err = store.ModelPlanCollectionWithCRTDLS(logger, false)
+		modelPlans, err = ModelPlansWithEchimpCRAndTDLS(echimpS3Client, viperConfig, logger, store)
```

Comment:
Since we're no longer using it, I'm just going to track deleting these `store` methods and the associated `sqlqueries` in MINT-3134!

Reply:
Thanks Clay!